src/entry.ts

```diff
@@ -90,6 +90,24 @@ class Entry {
         return this._data;
     }
 
+    /**
+    * 
+    * 
+    * Safely retrieves the value of a property from an object.
+    * 
+    * This function checks if the object has the specified property as its own property
+    * (i.e., not inherited from the prototype chain) before accessing it. This helps
+    * mitigate prototype pollution vulnerabilities.
+    * 
+    * @param {GenericObjectType} obj - The object from which to retrieve the property.
+    * @param {string | number} key - The key of the property to retrieve.
+    * @returns {any} - The value of the property if it exists, otherwise undefined.
+    */
+   
+    getPropertySafely(obj: GenericObjectType, key: string | number) {
+        return Object.prototype.hasOwnProperty.call(obj, key) ? obj[key] : undefined;
```

Comment:
Is this same as that of obj. hasOwnProperty(key)? What is the deference?

Reply:
This is a utility handler. So, here no object properties are inherited, and it would take `obj`, i.e., value and `key` from forEach to work in the context and update `value` accordingly.
This is checked and working as expected